package.json

```diff
@@ -78,7 +78,7 @@
     "react-portal": "^4.0.0-rc.0",
     "react-redux": "6.0.1",
     "react-router": "^5.0.0",
```

Comment:
Bump `react-router` at the same time?

Reply:
Smart 👍

Reply:
Bumpe react router?
![image](https://user-images.githubusercontent.com/7977650/89125891-d8e77900-d4e1-11ea-954f-62e31357cf95.png)

Reply:
😂

Reply:
You ready??
![image](https://user-images.githubusercontent.com/23152018/89125929-15b37000-d4e2-11ea-958b-24d8cea105e6.png)

Reply:
I think they will change it all back to the way it was in v3, so we need to rewrite everything again :upside_down_face:

Reply:
Upon closer inspection, it looks like they have finally gotten their shit together and implemented a lot of features, so it is actually improving on v3 in some ways. Perhaps it will not be as much of a PITA this time :crossed_fingers: